samples/shields/npm2100_ek/doc/index.rst

```diff
@@ -0,0 +1,120 @@
+.. zephyr:code-sample:: npm2100_ek
+   :name: nPM2100 EK
+
+   Interact with the nPM2100 PMIC using the EK buttons and the shell interface.
+
+Overview
+********
+
+This sample is provided for evaluation of the :ref:`npm2100_ek`.
+The sample show-cases the use of nPM2100 GPIO pins and the
+shell interface to control its features, including:
+
+- Regulators (BOOST, LDOSW)
+- GPIO
+
+Requirements
+************
+
+Make the following connections on the nPM2100 EK:
+
+- Remove all existing connections.
```

Comment:
Use * instead of - in the bullet lists.

Reply:
There is no such requirement - thanks!
https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#lists